build-system/tasks/performance/analytics-handler.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+function setupAnalyticsHandler(handlersList, handlerOptions, resolve) {
+  const {extraUrlParam} = handlerOptions;
+  // Q: Is the idea here that we might use more than 1 kv pair?
```

Comment:
No, we just needed a specific static identifier to tack onto the analytics request so that we would know when we intercepted it.

Reply:
thanks.